### ISSUE TITLE
Chem | Widgets | Structural Alerts: progress indicator is not  used

### ISSUE THREAD

**Issue by ahadzhanian** [datagrok-ai/public#1406]
Platform: Public Version: 1.11.1 Chem 1.4.8

Data: any, for example, v3000_sample
To reproduce:

1. Click a cell with a molecule
2. Go to **Property Pane > Structural Alerts**

**Expected result**: progress indicator is used

**Actual result**:  progress indicator is not used

![Image](https://user-images.githubusercontent.com/111368431/209655086-9b5cad3a-f2b4-452d-977c-8306d8ac520e.gif)

**Comment by vdyma** [comment]
@ahadzhanian thanks for reporting this!
I just tried it and it seems that the molecule 3 of this dataset is suppressing the progress indication somehow, but it seems it's not a problem actually. I noticed that Structural Alerts panel takes a lot of time to provide the result and freezes the page. It's related to https://github.com/datagrok-ai/public/issues/987. Once Structural Alerts algorithm is optimized this issue will also be resolved.
